iset.mm

```diff
@@ -155588,6 +155642,514 @@ the additive structure must be abelian (see ~ ringcom ), care must be
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Subspaces and spans in a left module
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c LSubSp $.
+
+  $( Extend class notation with linear subspaces of a left module or left
+     vector space. $)
+  clss $a class LSubSp $.
+
+  ${
+    $d a b j s x w $.
+    $( Define the set of linear subspaces of a left module or left vector
+       space.  (Contributed by NM, 8-Dec-2013.) $)
```

Comment:
Maybe this definition should be explained a little bit more: "A linear subspace of a left module or left vector space is an inhabited (in contrast to non-empty for non-intuitionistic logic) subset of the base set of the left-module/vector space with a closure condition on vector addition and scalar multiplication."

Reply:
Done.